pkg/hostagent/hostagent.go

```diff
@@ -164,6 +166,11 @@ func New(instName string, stdout io.Writer, signalCh chan os.Signal, opts ...Opt
 	limayaml.FillPortForwardDefaults(&rule, inst.Dir)
 	rules = append(rules, rule)
 
+	env := os.Getenv("LIMA_GRPC_PORT_FORWARDER")
```

Comment:
This should be parsed with `strconv.ParseBool`

Reply:
Done.